pephub/routers/api/v1/namespace.py

```diff
@@ -8,6 +8,8 @@
 from pepdbagent.exceptions import ProjectUniqueNameError
 from pepdbagent.const import DEFAULT_LIMIT_INFO
 from pepdbagent.models import ListOfNamespaceInfo
+from typing import Literal
+from typing_extensions import Annotated
```

Comment:
Drop [python < 3.9](https://fastapi.tiangolo.com/python-types/#type-hints-with-metadata-annotations) support? 🤔

Reply:
End of life for Python 3.8 in one year, so I would say let's leave it like that